tests/utils.py

```diff
@@ -558,8 +558,7 @@ def poll_cq(cq, count=1, data=None):
         for wc in tmp_wcs:
             if wc.status != e.IBV_WC_SUCCESS:
                 raise PyverbsRDMAError('Completion status is {s}'.
-                                       format(s=wc_status_to_str(wc.status)),
-                                       wc.status)
+                                       format(s=wc_status_to_str(wc.status)))
```

Comment:
you're right, that's confusing.
But this change, without further adjustments, will break some tests.
For example, in test_qpex there's the following check:
```
        try:
            u.poll_cq(client.cq)
        except PyverbsRDMAError as ex:
            if ex.error_code != e.IBV_WC_REM_ACCESS_ERR:
                raise ex
```
You may also find similar cases in some bad flow tests.

Reply:
One solution is to generalize the error string by passing "strerror_func" to  PyverbsRDMAError which translates the error code to a string. The os.strerror function would stay the default (converting errno->errno str).

Reply:
> But this change, without further adjustments, will break some tests.

Yes, you are right, my mistake.

Reply:
> Sorry, I didn't get you. do you mean we need to change the original PyverbsRDMAError ? Actually, I don't want to change the original PyverbsRDMAError class.

I meant to extend it, so it would take strerror_func as a parameter (in the initialization, defaulted to os.strerror).
Create a WC status to str function and pass it when raising that exception.
This is a more general solution that could be also used in other similar cases.

> Instead, I introduced a _poll_cq for tests, _poll_cq will return the error WC or the all successful WCs. So if the tests that want to check the wc.status, they should use _poll_cq instead.

I can stick with this for now. I'll extend the PyverbsRDMAError in the future if needed.